core-test/src/test/scala/org/bitcoins/core/crypto/MnemonicCodeTest.scala

```diff
@@ -0,0 +1,951 @@
+package org.bitcoins.core.crypto
+
+import org.bitcoins.core.gen.CryptoGenerators
+import org.bitcoins.core.util.BitcoinSUnitTest
+import org.scalacheck.Prop
+import org.scalatest.Assertion
+import scodec.bits.{BitVector, ByteVector, HexStringSyntax, BinStringSyntax}
+
+import scala.util.Try
+
+/**
+  * [[https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#test-vectors Trezor]]
+  * test vectors
+  */
+class MnemonicCodeTest extends BitcoinSUnitTest {
+  behavior of "MnemonicCode"
```

Comment:
Since this is new code, let's override the generatorDriveConfig to get more iterations on the property based tests. The old version only does 10 iterations, while the "new code" config does 100 iterations. You can do this like this:

```scala
  override implicit val generatorDrivenConfig: PropertyCheckConfiguration = {
    generatorDrivenConfigNewCode
  }
```